src/sdk/model/vm/vm.ts

```diff
@@ -1295,6 +1295,23 @@ export class Vm extends Entity implements EntityWithPerfSamples {
       return new Blob([response.data], { type: 'application/zip' });
     });
   }
+
+  /**
+   * Get an Advanced Backups download bundle link
+   * @param taskUuid {string} the task uuid used to create bundle
+   * @param fileName {string} optional file name
```

Comment:
Just double checking since it's new. This is an option that the user can supply as the download's file name?

Reply:
Okay looks like the content-disposition is set on the server side so I had to take this option off. According to the MDN, if the Content-Disposition header provides a different filename than one outlined in the "download" attribute or "filename", the Content-Disposition will take precedence